src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -1454,6 +1474,14 @@ PeerImp::onMessage (std::shared_ptr <protocol::TMGetObjectByHash> const& m)
     if (packet.query ())
     {
         // this is a query
+        if (send_queue_.size() >= Tuning::dropSendQueue)
```

Comment:
Nit: Rm the `// this is a query comment`. The `if` state immediately above says the same thing. (sometime I put such comments in `else` statements far away from the `if`)